### PR TITLE
Finalize and expose audits for homescreen & theme-color. 

### PR DESCRIPTION
**Pull request by paulirish**
image

image

### CODE REVIEW

File: aggregators/will-get-add-to-homescreen-prompt/index.js [hunk 1]
```diff
@@ -18,23 +18,48 @@
...
-      AddToHomescreen.criteria,
-      MobileFriendly.criteria
-    );
+    const manifestExists = require('../../audits/manifest/exists').name;
+    const manifestBackgroundColor = require('../../audits/manifest/background-color').name;
+    const manifestIcons = require('../../audits/manifest/icons').name;
+    const manifestIcons192 = require('../../audits/manifest/icons-192').name;
+    const manifestShortName = require('../../audits/manifest/short-name').name;
+
+    const criteria = {};
```
Comment (brendankenny) [inline aggregators/will-get-add-to-homescreen-prompt/index.js]: can get ES2015-fancy here and do
```
const criteria = {
  [manifestExists] = {
    value: true,
    weight: 1
  };

  [manifestBackgroundColor] = {
    value: true,
    weight: 1
  };
...
};
```

File: aggregators/will-get-add-to-homescreen-prompt/index.js [hunk 2]
```diff
@@ -25,20 +25,39 @@ class AddToHomescreen extends Aggregate {
     return 'Will Get Add to Homescreen Prompt';
   }
 
+  /**
+   * For the install-to-homescreen / install-app-banner prompt to show,
+   * Chrome needs the following:
+   *   - has a registered SW
```
Comment (brendankenny) [inline aggregators/will-get-add-to-homescreen-prompt/index.js]: is a SW necessary?

File: aggregators/omnibox-is-themed/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  static get name() {
+    return 'Omnibox Matches Brand Colors';
+  }
+
+  /**
+   * For the omnibox to adopt a theme color, Chrome needs the following:
+   *   - has valid manifest
+   *   - valid theme_color in manifest
+   *   - valid theme-color <meta> element
```
Comment (brendankenny) [inline aggregators/omnibox-is-themed/index.js]: do these need to match?

File: aggregators/will-get-add-to-homescreen-prompt/index.js [hunk 4]
```diff
@@ -18,23 +18,67 @@
...
+   *   - has valid manifest
+   *   - valid start_url
+   *   - valid name
+   *   - valid short_name
+   *   - icon of size >= 144x144 and png (either type `image/png` or filename ending in `.png`
+   * More details: https://github.com/GoogleChrome/lighthouse/issues/23
+   *
+   * TODO: We should allow icons >=144, rather than >= 192
```
Comment (brendankenny) [inline aggregators/will-get-add-to-homescreen-prompt/index.js]: we shouldn't close #23 until this is fixed. The change shouldn't be long in the icons-192 audit. Split on `'x'` (which is required to be lowercase after parsing), `parseInt` on each, then check that both width and height are >= 144

Comment (brendankenny) [inline aggregators/will-get-add-to-homescreen-prompt/index.js]: feel free to punt to another PR, we just shouldn't close the issue :)

**Comment by brendankenny** [comment]
reviewed!

File: cli.js [hunk 5]
```diff
@@ -35,7 +35,7 @@ const cli = meow(`
 
 `);
 
-const defaultUrl = 'https://m.flipkart.com';
+const defaultUrl = 'https://operasoftware.github.io/pwa-list/';
```
Comment (samccone) [inline cli.js]: eh?

**Comment by samccone** [comment]
This looks mostly good, once @brendankenny's comments are addressed we can :airplane: land this.

File: cli.js (hunk 5 above)
Comment (brendankenny) [inline cli.js]: yeah, is this just a nice example?

Comment (samccone) [inline cli.js]: was wondering why we were changing, but i know why now.

Comment (brendankenny) [inline cli.js]: care to share? :P

Comment (samccone) [inline cli.js]: because the manifest fetching is broken for flipkart.

Comment (brendankenny) [inline cli.js]: ah, got it

File: aggregators/will-get-add-to-homescreen-prompt/index.js (hunk 1 above)
Comment (paulirish) [inline aggregators/will-get-add-to-homescreen-prompt/index.js]: ah that's cool. i'll hold off for now though. makes things a bit more copypasteable.

File: aggregators/will-get-add-to-homescreen-prompt/index.js (hunk 2 above)
Comment (paulirish) [inline aggregators/will-get-add-to-homescreen-prompt/index.js]: yup. https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android?hl=en

File: aggregators/will-get-add-to-homescreen-prompt/index.js (hunk 4 above)
Comment (paulirish) [inline aggregators/will-get-add-to-homescreen-prompt/index.js]: sg.

File: aggregators/omnibox-is-themed/index.js (hunk 3 above)
Comment (paulirish) [inline aggregators/omnibox-is-themed/index.js]: Don't know. We'll have to ask mounir.

**Pull request merged**